src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -253,7 +253,7 @@ translation `p+L`, where `p` is only unique modulo `L`. The return type is a
 dict, the key `:base_points` gives an iterator over such `p`, and the key
 `:lineality_basis` lets one access a basis for the lineality space `L` of `PC`.
 
-See also [`vertices`](@ref) and [`lineality_space`](@ref).
+See also [`vertices(as::Type{PointVector{T}}, PC::PolyhedralComplex{T}) where {T<:scalar_types}`](@ref) and [`lineality_space`](@ref).
```

Comment:
```suggestion
See also [`vertices(PointVector{FieldElem}}, Polyhedron{FieldElem}})`](@ref) and [`lineality_space`](@ref).
```
I think this should also work. Not sure if it looks nicer though. It is both ugly.